// mirrorfly-uikit/dist/assets/images/index.js
export { default as Banner } from "./login-bg.png";
export { default as loader } from "./loader.svg";
export { default as ImgLoaderWhite } from "./loader-white.svg";
export { default as ImgFavicon } from "./favicon.png";
export { default as logo } from "./logo.png";
export { default as NewLogo } from "./new-images/logo.png";
export { default as NewLogoSm } from "./new-images/logo-sm.png";
export { default as NewLogoVector } from "./new-images/logo.svg";
export { default as ScanQRLogo } from "./scan-code-middle.png";
export { default as loaderSVG, default as loaderGIF } from "./loader.svg";
export { default as broadcast } from "./broadcast.png";
export { default as Loader } from "./loader.svg";
export { default as GroupChatAvatarImg } from "./sample-group-profile.svg";
export { default as SingleChatAvatarImg } from "./sample-callee-profile.png";
export { default as mediaAudio } from "./media-audio.svg";
export { default as mediaFile } from "./media-document.svg";
export { default as mediaimg1 } from "./media-img-1.png";
export { default as mediaimg2 } from "./media-img-2.png";
export { default as mediaimg3 } from "./media-img-3.png";
export { default as mediaVideo } from "./media-video.svg";
export { default as AudioRecord } from "./audio-record.png";
export { default as AudioFile } from "./audio-file.png";
export { default as Document } from "./doc-thumb.png";
export { default as audioFileImg } from "./audio-file-white.svg";
export { default as audioRecordImg } from "./recorder.svg";
export { default as ChatContactImg } from "./avatar.png";
export { default as Excel } from "./media-types/excel.svg";
export { default as Pdf } from "./media-types/pdf.svg";
export { default as Ppt } from "./media-types/ppt.svg";
export { default as Placeholder } from "./media-types/placeholder.svg";
export { default as Txt } from "./media-types/txt.svg";
export { default as Word } from "./media-types/word.svg";
export { default as Rar } from "./media-types/rar.svg";
export { default as Zip } from "./media-types/Zip.svg";
export { default as Csv } from "./media-types/csv.svg";
export { default as AudioMessage } from "./reply-icons/audioMessage.svg";
export { default as ReplyCamera } from "./reply-icons/reply-camera.svg";
export { default as ReplyVideo } from "./reply-icons/reply-video.svg";
export { default as ChatContactSample } from "./sample-profile.svg";
export { default as SampleGroupProfile } from "./sample-group-profile.svg";
export { default as sampleChatProfile } from "./sample-profile.svg";
export { default as AttachAudio } from "./attachAudio.png";
export { default as dragDrop } from "./media-types/drag-drop.png";
export { default as ImgEmptyArchive } from "./emptyArchive.png";
export { default as GroupProfile } from "./sample-group-profile.svg";
export { default as SampleProfile } from "./sample-profile.svg";
export { default as ImgBgCallEnd } from "./webcall/bg-call-end.svg";
export { default as SampleProfile1 } from "./sarah.png";
export { default as Sarah1 } from "./sarah.png";
export { default as SampleProfile2 } from "./sarah1.jpg";
export { default as SampleImage } from "./seleced-files-1.png";
export { default as ImgPreview } from "./preview_image.png";
export { default as ImgEditProfile } from "./editProfile.svg";
export { default as ImgUserPlaceholder } from "./user-placeholder.svg";
export { default as MeetingLogo } from "./meeting_logo.png";
export { default as CallEndimg } from "./call-end.png";
export { default as profile1 } from "./profile1.png";
export { default as profile2 } from "./profile2.png";
export { default as profile3 } from "./profile3.png";
export { default as ImgSmile } from "./smile.svg";
export { default as ImgBgbubble } from "./box-layout/bg-bubble.svg";

// export {default as VideoPlaceholder} from '../videos/placeholder.mp4';

export { ReactComponent as IconSmileHoc } from "./smile.svg";
export { ReactComponent as CallLogsHoc } from "./callLog.svg";
export { ReactComponent as CallLogCountHoc } from "./callLogCount.svg";
export { ReactComponent as NewChatHoc } from "./newchat-new.svg";
export { ReactComponent as NewGroupHoc } from "./newgroup.svg";
export { ReactComponent as DotMenuHoc } from "./optionmenu-new.svg";
export { ReactComponent as SettingsHoc } from "./settings.svg";
export { ReactComponent as ChatMuteHoc } from "./chatMute.svg";
export { ReactComponent as ChatMuteOffHoc } from "./chatmute-off.svg";
export { ReactComponent as IconMeetingVideoHoc } from "./meeting-link-video.svg";
export { ReactComponent as IconMissedCallAudioHoc } from "./missedCallAudio.svg";
export { ReactComponent as IconMissedCallVideoHoc } from "./missedCallVideo.svg";
export { ReactComponent as LoginInfoHoc } from "./logininfo.svg";
export { ReactComponent as RejectCallHoc } from "./rejectCall.svg";
export { ReactComponent as AudioOnHoc } from "./webcall/audio-on.svg";
export { ReactComponent as AudioOffHoc } from "./webcall/audio-mute.svg";
export { ReactComponent as VideoOnHoc } from "./webcall/video-on.svg";
export { ReactComponent as VideoOffHoc } from "./webcall/video-off.svg";
export { ReactComponent as AttenCallHoc } from "./webcall/attenCall.svg";
export { ReactComponent as BackToChatHoc } from "./webcall/backToChat.svg";
export { ReactComponent as AvatarHoc } from "./sample-profile.svg";
// export { ReactComponent as AvatarCalleeHoc } from "./sample-callee-profile.png";
export { ReactComponent as ChatHoc } from "./new-images/chart.svg";
export { ReactComponent as CallHoc } from "./new-images/call.svg";
export { ReactComponent as UnsavedContactHoc } from "./call.svg";
export { ReactComponent as VideoHoc } from "./new-images/video.svg";
export { ReactComponent as ArrowBackHoc } from "./arrowback.svg";
export { ReactComponent as MailIconHoc } from "./mailicon.svg";
export { ReactComponent as MailiconSolidHoc } from "./mailiconSolid.svg";
export { ReactComponent as EmptyContactHoc } from "./new-images/start-chart.svg";
export { ReactComponent as SearchHoc } from "./search.svg";
export { ReactComponent as CloseHoc } from "./search_close.svg";
export { ReactComponent as LogoutHoc } from "./logout.svg";
export { ReactComponent as TickHoc } from "./Tick.svg";
export { ReactComponent as TickHoc2 } from "./tick2.svg";
export { ReactComponent as RecordAudioHoc } from "./record-mic.svg";
export { ReactComponent as StartRecordHoc } from "./startRecord.svg";
export { ReactComponent as StopRecordHoc } from "./stopRecord.svg";
export { ReactComponent as IconLoaderHoc } from "./loader.svg";
export { ReactComponent as ClosePopupHoc } from "./close.svg";
export { ReactComponent as FloatingCallVideoHoc } from "./floating-call-1.svg";
export { ReactComponent as FloatingCallAudioHoc } from "./floating-call-2.svg";
export { ReactComponent as FloatingCallActionHoc } from "./floating-call-action.svg";
export { ReactComponent as FloatingCallActionSmHoc } from "./floating-call-action-sm.svg";
export { ReactComponent as CallLogs2Hoc } from "./callLog2.svg";
export { ReactComponent as CloseWhiteHoc } from "./close-white.svg";
export { ReactComponent as RetakeHoc } from "./retake.svg";
export { ReactComponent as CameraIconHoc } from "./cameraicon.svg";
export { ReactComponent as MenuHoc } from "./arrowdown.svg";
export { ReactComponent as DropdownArrowHoc } from "./arrowdown-Dark.svg";
export { ReactComponent as DropdownArrow2Hoc } from "./arrowdown-2.svg";
export { ReactComponent as Close2Hoc } from "./close2.svg";
export { ReactComponent as InfoHoc } from "./errorinfo.svg";
export { ReactComponent as AddParticiapantsHoc } from "./addParticiapants.svg";
export { ReactComponent as BlockedHoc } from "./blocked.svg";
export { ReactComponent as DeleteHoc } from "./delete.svg";
export { ReactComponent as DoneHoc } from "./done.svg";
export { ReactComponent as EditHoc } from "./edit.svg";
export { ReactComponent as GoBackHoc } from "./goback.svg";
export { ReactComponent as StatusHoc } from "./status.svg";
export { ReactComponent as UnBlockHoc } from "./unBlock.svg";
export { ReactComponent as NextHoc } from "./next.svg";
export { ReactComponent as BlockedIconHoc } from "./blocked-white.svg";
export { ReactComponent as ZoomInHoc } from "./zoom-in.svg";
export { ReactComponent as ZoomOutHoc } from "./zoom-out.svg";
export { ReactComponent as ZoomRestoreHoc } from './zoom-restore.svg';
export { ReactComponent as AttachmentHoc } from "./attachments.svg";
export { ReactComponent as SendMessageHoc } from "./sendMessage.svg";
export { ReactComponent as ChatAudioReceiverHoc } from "./audio_record.svg";
export { ReactComponent as ChatAudioSenderHoc } from "./chataudio.svg";
export { ReactComponent as ChatAudioReceiver2Hoc } from "./audio-file.svg";
export { ReactComponent as ChatAudioSender2Hoc } from "./audio-file.svg";
export { ReactComponent as ChatAudioRecorderHoc } from "./recorder.svg";
export { ReactComponent as ChatAudioRecorderDarkHoc } from "./recorder-dark.svg";
export { ReactComponent as Audio2Hoc } from "./recentAudio.svg";
export { ReactComponent as ChatContactHoc } from "./chatcontact.svg";
export { ReactComponent as ChatContactReceiHoc } from "./chatcontactrecei.svg";
export { ReactComponent as ContactDetailsHoc } from "./contactDetails.svg";
export { ReactComponent as DocumentReceiverHoc } from "./doc.svg";
export { ReactComponent as DocumentSenderHoc } from "./docrecei.svg";
export { ReactComponent as DocumentDownloadHoc } from "./download-doc.svg";
export { ReactComponent as SingleForwardHoc } from "./singleForward.svg";
export { ReactComponent as SingleForward2Hoc } from "./forward-new.svg";
export { ReactComponent as PlaybtnHoc } from "./playbtn.svg";
export { ReactComponent as Playbtn2Hoc } from "./playbtn2.svg";
export { ReactComponent as VideoIconHoc } from "./videoicon.svg";
export { ReactComponent as VideoIcon3Hoc } from "./videoicon2.svg";
export { ReactComponent as AttachmentCameraHoc } from "./file-camera.svg";
export { ReactComponent as AttachmentDocHoc } from "./file-document.svg";
export { ReactComponent as AttachmentImageVideoHoc } from "./file-image.svg";
export { ReactComponent as AttachMusicHoc } from "./file-headset.svg";
// Message Dropdown option Icons start
export { ReactComponent as DeleteMessageHoc } from "./deleteMessage.svg";
export { ReactComponent as ForwardHoc } from "./forward.svg";
export { ReactComponent as MessageInfoHoc } from "./messageinfo.svg";
export { ReactComponent as ReplyHoc } from "./reply.svg";
export { ReactComponent as StarHoc } from "./starred.svg";
export { ReactComponent as StarredHoc } from "./starred.svg";
export { ReactComponent as UnStarHoc } from "./unstar.svg";
export { ReactComponent as StarredDarkHoc } from "./starred-dark.svg";
export { ReactComponent as UnStarDarkHoc } from "./unstar-dark.svg";
export { ReactComponent as TranslateHoc } from "./translate.svg";
// Message Dropdown option Icons end

// Media Preview Icons

export { ReactComponent as PreviewCloseHoc } from "./preview_close.svg";
export { ReactComponent as PreviewDownloadHoc } from "./preview_download.svg";
export { ReactComponent as PreviewForwardHoc } from "./preview_forward.svg";
export { ReactComponent as PreviewStarHoc } from "./preview_star.svg";
export { ReactComponent as PreviewStarActiveHoc } from "./preview_star_active.svg";
export { ReactComponent as DropDownloadHoc } from "./drop-download.svg";
export { ReactComponent as PreviewChatHoc } from "./preview_chat.svg";

// Media Preview Icons end

export { ReactComponent as SelectedForwardHoc } from "./message-forward.svg";
export { ReactComponent as SelectedCloseHoc } from "./selected-close.svg";
export { ReactComponent as ReplyAudioHoc } from "./reply-icons/reply-audio.svg";
export { ReactComponent as ReplyAudioRecordedHoc } from "./reply-icons/reply-audio-recorded.svg";
export { ReactComponent as ReplyDocumentHoc } from "./reply-document.svg";
export { ReactComponent as ReplyContactHoc } from "./reply-icons/reply-contact.svg";
export { ReactComponent as ReplyPlayHoc } from "./reply-icons/reply-play.svg";
export { ReactComponent as ChatAudioSenderIconHoc } from "./audio_record.svg";
export { ReactComponent as ChatAudioReceiverIconHoc } from "./chataudio.svg";
export { ReactComponent as ChatBubbleHoc } from "./chatBubble.svg";
export { ReactComponent as FiletypePlaceholderSenderHoc } from "./media-types/filetype-placeholder.svg";
export { ReactComponent as FiletypePlaceholderReceiverHoc } from "./media-types/filetype-placeholderReceiver.svg";
export { ReactComponent as AudioHoc } from "./audio.svg";
export { ReactComponent as CameraHoc } from "./camera.svg";
export { ReactComponent as CloseReplyHoc } from "./close-reply.svg";
export { ReactComponent as ContactHoc } from "./contact-icon.svg";
export { ReactComponent as DocumentIconHoc } from "./document.svg";
export { ReactComponent as LocationHoc } from "./location-icon.svg";
export { ReactComponent as ReplyVcardHoc } from "./reply-vcard.svg";
export { ReactComponent as VideoIcon2Hoc } from "./video.svg";
export { ReactComponent as ResetHoc } from "./reset.svg";
export { ReactComponent as AudioCallHoc } from "./audioCall.svg";
export { ReactComponent as VideoCallHoc } from "./videoCall.svg";
export { ReactComponent as CloseMessageInfoDetailsHoc } from "./closeMessageInfoDetails.svg";
export { ReactComponent as EmptyDeliverHoc } from "./emptyDeliver.svg";
export { ReactComponent as IconEmptyArchiveHoc } from "./emptyArchive.svg";
export { ReactComponent as OpenMessageInfoDetailsHoc } from "./openMessageInfoDetails.svg";
export { ReactComponent as CloseMessageInfoHoc } from "./closeMessageInfo.svg";
export { ReactComponent as CreateGroupHoc } from "./createGroup.svg";
export { ReactComponent as EditCameraHoc } from "./editcamera.svg";
export { ReactComponent as RemoveHoc } from "./remove.svg";
export { ReactComponent as TakephotoHoc } from "./takephoto.svg";
export { ReactComponent as UploadHoc } from "./upload.svg";
export { ReactComponent as Upload2Hoc } from "./upload2.svg";
export { ReactComponent as Upload3Hoc } from "./uploading2.svg";
export { ReactComponent as ViewphotoHoc } from "./view.svg";
export { ReactComponent as BlockedInfoHoc } from "./blockedInfo.svg";
export { ReactComponent as RechatchatimageHoc } from "./new-images/start-chart.svg";
export { ReactComponent as NochatHoc } from "./nochat.svg";
export { ReactComponent as AudioOnGoingCallHoc } from "./incoming-calls.svg";
export { ReactComponent as OnGoingCallHoc } from "./onGoingCall.svg";
export { ReactComponent as RecentCallHangupHoc } from "./webcall/recent-call-hangup.svg";
export { ReactComponent as RecentCallMuteHoc } from "./webcall/recent-call-mute.svg";
export { ReactComponent as RecentCallMuteActiveHoc } from "./webcall/recent-call-mute-active.svg";
export { ReactComponent as RecentVideoCallHoc } from "./webcall/recent-call-video.svg";
export { ReactComponent as RecentAudioCallHoc } from "./webcall/recent-call-audio.svg";
export { ReactComponent as RecentDeletedMessageHoc } from "./recent_deleted.svg";
export { ReactComponent as TileViewHoc } from "./webcall/tileView.svg";
export { ReactComponent as EmptyCallLogHoc } from "./webcall/empty-calllog.svg";
export { ReactComponent as TileViewRemoveHoc } from "./webcall/tileViewRemove.svg";
export { ReactComponent as IconIncomingHoc } from "./webcall/webcall-incoming.svg";
export { ReactComponent as IconOutgoingHoc } from "./webcall/webcall-outgoing.svg";
export { ReactComponent as IconIncomingMissedHoc } from "./webcall/missedcall-incoming.svg";
export { ReactComponent as IconAudiocallHoc } from "./webcall/webcall-audio.svg";
export { ReactComponent as IconVideocallHoc } from "./webcall/webcall-video.svg";
export { ReactComponent as IconPinHoc } from "./webcall/pin.svg";
export { ReactComponent as IconPinActiveHoc } from "./webcall/pin-active.svg";
export { ReactComponent as StarBlueHoc } from "./star-blue.svg";
export { ReactComponent as StarGrayHoc } from "./star-gray.svg";
export { ReactComponent as InfoIconHoc } from "./info.svg";
export { ReactComponent as EmptySearchHoc } from "./new-images/emptySearch.svg";
export { ReactComponent as EventWaitingHoc } from "./loader.svg";
export { ReactComponent as MediaDownloadHoc } from "./media-download.svg";
export { ReactComponent as MediaDownloadLiteHoc } from "./media-download-lite.svg";
export { ReactComponent as NoMediaHoc } from "./no-media.svg";
export { ReactComponent as AudioPermissionIconHoc } from "./new-images/audioPermission.svg";
export { ReactComponent as ClearChatHoc } from "./clearChat.svg";
export { ReactComponent as FileLoaderCircleHoc } from "./file-loader-close.svg";

export { ReactComponent as FileloaderHoc } from "./file-loader.svg";
export { ReactComponent as IconUploadingHoc } from "./uploading.svg";
export { ReactComponent as IconSendingFailedHoc } from "./sendingFailed.svg";
export { ReactComponent as IconBackArrowHoc } from "./back-arrow.svg";
export { ReactComponent as IconScroltToDownHoc } from "./scrol-to-down.svg";
export { ReactComponent as IconExitGroupHoc } from "./exitgroup.svg";
export { ReactComponent as IconInviteHoc } from "./invite.svg";
export { ReactComponent as IconParticiantsHoc } from "./particiants.svg";
export { ReactComponent as IconStaredArrowHoc } from "./stared-arrow.svg";
export { ReactComponent as IconStaredMenuHoc } from "./optionmenu-new-white.svg";
export { ReactComponent as IconNoStaredMessageHoc } from "./noStaredMessage.svg";
export { ReactComponent as IconNoStarHoc } from "./noStar.svg";
export { ReactComponent as IconCopyHoc } from "./copy.svg";
export { ReactComponent as IconLinkHoc } from "./link.svg";
export { ReactComponent as IconAudioIncommingCallHoc } from "./webcall/audioIncommingCall.svg";
export { ReactComponent as IconAudioMissedCallHoc } from "./webcall/audioMissedCall.svg";
export { ReactComponent as IconAudioOutgoingCallHoc } from "./webcall/audioOutgoingCall.svg";
export { ReactComponent as IconVideoOutgoingCallHoc } from "./webcall/VideoOutgoingCall.svg";
export { ReactComponent as IconVideoIncommingCallHoc } from "./webcall/videoIncommingCall.svg";
export { ReactComponent as IconVideoMissedCallHoc } from "./webcall/videoMissedCall.svg";
export { ReactComponent as IconInvalidLinkHoc } from "./webcall/invalid-link.svg";
export { ReactComponent as IconEditProfileHoc } from "./editProfile.svg";
export { ReactComponent as IconUserPlaceholderHoc } from "./user-placeholder.svg";
export { ReactComponent as BackToChatWhiteHoc } from "./webcall/backToChatWhite.svg";
export { ReactComponent as CallEndiconHoc } from "./callEndicon.svg";
export { ReactComponent as CallendiconplainHoc } from "./call-ended.svg";
export { ReactComponent as IconVerifiyedAccountHoc } from "./verifiyed_account.svg";
export { ReactComponent as IconReportHoc } from "./report.svg";
export { ReactComponent as IconGroupReportHoc } from "./group_report.svg";
export { ReactComponent as IconSingleReportHoc } from "./single_report.svg";
export { ReactComponent as IconDarkThemeHoc } from "./darktheme/dark-theme.svg";
export { ReactComponent as IconLightThemeHoc } from "./darktheme/light-theme.svg";
export { ReactComponent as IconArchiveHoc } from "./archive.svg";
export { ReactComponent as  CopyTextHoc} from "./copyText.svg";
